src/test/java/org/mitre/synthea/export/BB2RIFExporterTest.java

```diff
@@ -45,9 +47,32 @@
 import org.mitre.synthea.helpers.Utilities;
 import org.mitre.synthea.world.agents.Person;
 import org.mitre.synthea.world.concepts.Claim;
-import org.mitre.synthea.world.concepts.HealthRecord;
 
 public class BB2RIFExporterTest {
+
+  public class MockMapper extends BB2RIFExporter.CodeMapper {
+    public MockMapper() {
+      super("MOCK_MAPPING_FILE_THAT_DOES_NOT_EXIST.JSON");
+    }
+
+    public boolean canMap(String codeToMap) {
+      return true;
+    }
+
+    public boolean hasMap() {
+      return true;
+    }
+
+    public String map(String codeToMap, String bfdCodeType, RandomNumberGenerator rand,
```

Comment:
I got a checkstyleTest warning about this function missing a Javadoc comment.

Reply:
Should be fixed now.